cmd/admin-handlers.go

```diff
@@ -801,6 +810,91 @@ func convertValueType(elem []byte, jsonType gjson.Type) (interface{}, error) {
 	}
 }
 
+func isInvalidJSONField(expectedKeys gjson.Result, elem []byte) (string, string) {
+	keys := string(elem)
+	var found bool
+	var fullKeyName string
+	// Validation for a single key
+	if expectedKeys.Type == gjson.Null &&
+		expectedKeys.Raw == "" &&
+		expectedKeys.Str == "" &&
+		expectedKeys.Num == 0 &&
+		expectedKeys.Index == 0 {
+		return "Invalid key", ""
+	}
+	// Validation for multiple key names in JSON format
+	var skipChrsFront int
+	var skipChrsEnd int
+	if expectedKeys.Type == gjson.JSON {
+		if len(keys) == 2 {
+			return "Invalid/Empty JSON data", ""
+		}
+		raw := expectedKeys.Raw
+		if keys[:1] == "{" {
+			skipChrsFront = 1
+		} else {
+			skipChrsFront = 0
+		}
+		if keys[len(keys)-1:] == "}" {
+			skipChrsEnd = 1
+		} else {
+			skipChrsEnd = 0
+		}
+		r := regexp.MustCompile(`[^\s":]+|"([^"]*)"`)
```

Comment:
needs comment on what regex is looking for.

Reply:
also the regex should be globally defined...

Reply:
OK